Update several xrefs and syntax issues

cthoyt opened the pull request:
This PR solves several logical and syntactic issues with references being encoded improperly

addresses most of #7

cthoyt committed:
Remove dummy xrefs

cthoyt committed:
Update sibo-edit.obo

cthoyt committed:
Update sibo-edit.obo

matentzn commented on sibo-edit.obo:
```diff
@@ -839,19 +839,19 @@ is_a: SIBO:0000149 ! Anatomical part
 id: SIBO:0000142
 name: presclerite
-def: "Presclerite- A distinctly differentiated anterior section of an abdominal sclerite, separated from the remainder of the sclerite by a ridge, constriction, or both. In the ant abdomen it is usual for the posterior portion of each sclerite to overlap the anterior portion of the following segment. The overlapped area usually lacks sculpture and pilosity, but the absence of these features alone ...
+def: "Presclerite- A distinctly differentiated anterior section of an abdominal sclerite, separated from the remainder of the sclerite by a ridge, constriction, or both. In the ant abdomen it is usual for the posterior portion of each sclerite to overlap the anterior portion of the following segment. The overlapped area usually lacks sculpture and pilosity, but the absence of these features alone ...
```
You left the bibliographic reference - wont that alone create a wrong xref?

matentzn committed:
Reformat sibo-edit to adhere to new OBO format standards

A lot of edits happened here. @cthoyt, I normalised natural language bibliographic references as part of the definition string, and fixed a few syntactic errors.

matentzn committed:
Further normalisation of edit file

matentzn committed:
reserialise OBO and RDF/XML format

Lots of diff, but they all come from the latest edit to the edit file

cthoyt committed:
Update sibo-edit.obo

cthoyt commented:
thanks @matentzn. LGTM
